UDP/course_materialized_view.md

```diff
@@ -22,6 +22,9 @@ on pe.person_id = cse.person_id join entity.person p on p.person_id = cse.person
 courses_enrollment as (select cst.*, e.person_id, e.uniqname, e.person_name from courses_sections_of_current_term cst join enrollment e on e.course_section_id = cst.course_section_id),
 course_grades as (select ce.*, cg.le_current_score as "Current_Grade", cg.le_final_score as "Final_Grade", cg.gpa_cumulative_excluding_course_grade as cumulative_gpa from entity.course_grade cg join courses_enrollment ce on ce.course_section_id = cg.course_section_id and ce.person_id = cg.person_id),
 
+avg_course_grade as (select "Canvas_Course_ID", round(AVG("Current_Grade"),2) as avg_course_grade  from course_grades group by  "Canvas_Course_ID"),
```

Comment:
@zqian this information about overall COURSE Grade i.e `current_grade` is already available from one of the column (From before) and we never checked if this student grade is release or posted. 

Is this new? since what I am doing is now is just calculating the average grade from the available list of grades.

Reply:
According to the [documentation](https://docs.udp.unizin.org/ucdm/data-dictionary-2.0.html#course_grade) from Unizin `

le_current_score | The most recently calculated and visible score.
-- | --


I think we are only pulling the grade that is visible to student

There are other column values that show hidden grades like these 


le_hidden_current_score | The most recently calculated score that is not yet visible.
-- | --
le_hidden_final_score | The final calculated score that is not yet visible.

Reply:
Just checked. The current_grade is based on the `le_current_score` field inside UDP's `course_grade` table, which by definition is "The most recently calculated and visible score."

So not an issue indeed.